services/device-registry-base/src/main/java/org/eclipse/hono/service/management/tenant/TrustedCertificateAuthority.java

```diff
@@ -206,12 +227,22 @@ public final byte[] getPublicKey() {
      */
     @JsonProperty(TenantConstants.FIELD_PAYLOAD_CERT)
     public final TrustedCertificateAuthority setCertificate(final byte[] certificate) throws CertificateException {
-
         final CertificateFactory factory = CertificateFactory.getInstance("X.509");
         cert = (X509Certificate) factory.generateCertificate(new ByteArrayInputStream(certificate));
         return this;
     }
 
+    /**
+     * Gets the trusted certificate.
+     *
+     * @return The certificate in encoded form.
+     * @throws CertificateEncodingException if certificate cannot be encoded.
+     */
+    @JsonProperty(TenantConstants.FIELD_PAYLOAD_CERT)
+    public final byte[] getCertificate() throws CertificateEncodingException {
+        return cert == null ? null : cert.getEncoded();
+    }
+
```

Comment:
what is this method required for?

Reply:
It is just used to include the persisted trusted certificate in tenant GET request. But if we decide to always use [ocspResponderCert](https://github.com/eclipse-hono/hono/blob/73472a4ee8a7c235cbee998f67a4f61b06cbaf70/services/device-registry-base/src/main/java/org/eclipse/hono/service/management/tenant/TrustedCertificateAuthority.java#L94) instead (as described in next comment), I can just remove it.